setup.cfg

```diff
@@ -31,7 +31,8 @@ classifiers=
 zip_safe = true
 python_requires= >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
 install_requires=
-    setuptools
+    setuptools; python_version < "3.8"
+    packaging; python_version >= "3.8"
```

Comment:
since  the intended use-case is with setuptools as of now, i dont see any demonstrated reason to bring in packaging, as far as i can tell all it does is break things

before packaging can/should be considered, there should be a tightening of the supported setuptools version in install_requires

Reply:
So what do you suggest? Pinning the packaging version to what setuptools uses?

Reply:
Basically until setuptools drops the vendoring we should not add packaging as dependency

Reply:
So what do you suggest?

Reply:
Migration of the version class specification from the module to passing a callable to the configuration

Then the setuptools integration can pass it in and other new integrations could pass the packaging on sooner

Reply:
I'll provide a better example when I get back home